Automattic/wp-calypso
# SEO Tools: Fix upgrade nudge for Jetpack sites

The new Jetpack plans didn't include the SEO preview tools feature, and
so we would display a nudge to upgrade to a business plan. Looking at
the legacy plans, it seems as though this feature was available on all
paid Jetpack plans, not just business.

Testing with a premium WPCOM plan, the meta section isn't shown, which
is the same for free Jetpack sites. Therefore it would appear that we
don't display this nudge in this context.

#### Changes proposed in this Pull Request

This change adds the feature to the new Jetpack plans and updates the
nudge to prompt people with Jetpack sites to upgrade to Security Daily.

From reading that is the equivalent of the Premium plan, but it would be 
good to get that business logic confirmed. 

#### Testing instructions

The testing instructions are the same as #48841.

* Create a Jetpack site ~with one of the new Security or Complete plans~ ([after some discussion with @kraftbj ](https://github.com/Automattic/wp-calypso/issues/48841#issuecomment-772913611)we've decided to open this feature up to all Jetpack sites)
* Go to /tools/marketing/traffic
* Make sure that the "Enable SEO Tools to optimize your site for search engines" option is enabled
* In the meta section click on Show Preview, and you'll be prompted to upgrade to a business plan
* With this PR you should see the preview.

Fixes #48841
